style.css

```diff
@@ -1,10 +1,257 @@
-body {
-background-color: grey;
-font-family: "GothamPro";
+.body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  align-content: center;
 }
-
-.item__price {
-font-weight: bold;
+.container {
+  max-width: 1030px;
+  margin-left: 0;
 }
-
- 
+.header {
+  max-height: 96px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 20px;
+}
+.header-menu,
+.header-menu__right {
+  display: flex;
+  margin-top: 28px;
+  justify-content: space-between;
+}
+.header-menu__logo {
+  width: 55px;
+  height: 55px;
+  margin-right: 45px;
+  margin-top: 10px;
+}
+.header-menu__nav,
+.header-menu__right {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.header-menu__nav-link, 
+.header-menu__right-link {
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #000;
+  font-weight: 700;
+  align-items: center;
+  margin-right: 35px;
+}
+.header-menu__nav-link:hover {
+  text-decoration: underline;
+}
+.header-menu__right-link:hover {
+  text-decoration: underline;
+}
+.header__form {
+  position: relative;
+  margin-top: 28px;
+  margin-left: 140px;
+  padding-right: 20px;
+}
+.header__input {
+  border: 1px solid;
+}
+.header-menu__link-cart {
+  vertical-align: middle;
+}
+.header-menu__moblogo {
+  display: none;
+}
+.header__burger-wrap {
+  display: none;
+}
+.content {
+  display: block;
+  width: 100%;
+  max-width: 1030px;
+  height: 280px;
+  background: url(./images/potter.png);
+  position: relative;
+  margin: 0 auto 67px auto;
+}
+.content__block {
+  display: flex;
+  flex-direction: column;
+  width: 209px;
+  padding-left: 54px;
+  padding-top: 82px;
+  justify-content: center;
+  align-items: flex-start;
+}
+.content__title {
+  font-size: 30px;
+  font-weight: bold;
+  line-height: 29px;
+  text-transform: uppercase;
+  color: #fff;
+  text-align: left;
+  position: relative;
+}
+.content__block-link {
+  justify-content: flex-start;
+}
+.link__jump {
+  font-size: 12px;
+  font-weight: 700;
+  text-transform: uppercase;
+  width: 78px;
+  text-decoration: none;
+  color: #fff; 
+  position: relative;
+}
+.arrow {
+  padding-left: 7px;
+  width: 45px;
+  display: inline-block;
+}
+.product {
+  margin: 0 auto 52px auto;
+}
+ .product__title {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step/pullRequest?prid=5998468)